.changeset/chilly-zebras-appear.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-anywidget": major
+---
+
+move package.json to root folder
```

Comment:
Also, the text that goes in this file will end up in the final CHANGELOG for `create-anywidget` when we make a release ([Example](https://github.com/manzt/anywidget/releases/tag/create-anywidget%400.1.2)), so I typically like to have these messages a bit more verbose.

```suggestion
feat: move `package.json` to template root directories

This makes it more ergonomic to install JavaScript dependencies and start the development server.
```

Reply:
awesome, thanks for your advices on package management!